### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by dsantos-lean**
Hello @CobeySmith, I apologize for only requesting this now. Can you look at my codes please? May need improvements on some parts. 

Thank you,

Lean

### CODE REVIEW

File: prac_06/guitar.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    def __init__(self, name="", year=0, cost=0.0, age=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+        self.age = age
+
+    def __str__(self):
+        return "{} ({}) : $ {}".format(self.name, self.year, self.cost)
+
+    def get_age(self, current_year=2022):
```
Comment (CobeySmith) [inline prac_06/guitar.py]: you can just do:
return 2022 - self.year

File: prac_06/guitars.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    guitars = []
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
```
Comment (CobeySmith) [inline prac_06/guitars.py]: No error checking for year and cost

**Comment by CobeySmith** [comment]
The cars program is pretty good. For guitars, add error checking when getting your values and there are a few places you could simplify your code further but are not needed.

File: prac_06/guitar.py (hunk 1 above)
Comment (dsantos-lean) [reply prac_06/guitar.py]: That looks better :)
..changed code to

def get_age(self):
        return 2022 - self.year

File: prac_06/guitars.py (hunk 2 above)
Comment (dsantos-lean) [reply prac_06/guitars.py]: you're right!
..added error checking function

**Pull request merged**